rotkehlchen/chain/optimism/node_inquirer.py

```diff
@@ -44,10 +46,11 @@ def __init__(
             contracts=EvmContracts[Literal[ChainID.OPTIMISM]](
                 chain_id=ChainID.OPTIMISM,
             ),
+            proxies_inquirer=proxies_inquirer,
             connect_at_start=connect_at_start,
             rpc_timeout=rpc_timeout,
         )
-        self.etherscan = cast(OptimismEtherscan, self.etherscan)
+        self.etherscan: OptimismEtherscan
```

Comment:
I don't know if @LefterisJP had a reason for this but maybe is cleaner. Cast is only for typing afaik and leaves the object unchanged during execution

Reply:
Yeah, I just got a warning from mypy here. And changing from `cast` to `: Type` resolved the warning